lib/static/modules/web-vitals.js

```diff
@@ -0,0 +1,13 @@
+import {getCLS, getFID, getFCP, getLCP, getTTFB} from 'web-vitals';
+
+const measurePerformance = onPerfEntry => {
+    if (onPerfEntry && onPerfEntry instanceof Function) {
+        getCLS(onPerfEntry);
+        getFID(onPerfEntry);
+        getFCP(onPerfEntry);
+        getLCP(onPerfEntry);
+        getTTFB(onPerfEntry);
```

Comment:
More info about all of these metrics can be found here - https://web.dev/vitals/